pyproject.toml

```diff
@@ -28,6 +28,12 @@ exclude_lines = [
 
 [tool.coverage.html]
 
+[tool.interrogate]
+fail-under = 26
```

Comment:
What does this mean?

Reply:
Interrogate will return a failure code on the command line if coverage is below 26% (mirroring how test coverage works).  We're currently just over that, so this is base level to start with!